maintainer.sh

```diff
@@ -1,16 +1,25 @@
 #!/bin/sh
-set -x
-
 aclocal # Set up an m4 environment
 autoconf # Generate configure from configure.ac
 automake --add-missing # Generate Makefile.in from Makefile.am
 
 #Test configure & build
-./configure --prefix=$PWD CC=clang
+./configure CC=clang
 make
-make install
+
+#install sgxtop / sgxstat link
+sudo make install
```

Comment:
I suspect that we should just skip the install altogether.  If someone wants to do a sudo, it should probably be explicit.

I think that most administrators would expect to do a separate "sudo make install" for the package.

Reply:
done